beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/hive/HiveMetadataCleaner.java

```diff
@@ -53,6 +62,11 @@ public boolean dropPartition(HousekeepingMetadata housekeepingMetadata) {
 
   @Override
   public boolean tableExists(String databaseName, String tableName) {
-    return client.tableExists(databaseName, tableName);
+    return hiveClient.tableExists(databaseName, tableName);
+  }
+
+  @Override
+  public void close(){
```

Comment:
Maybe this one could implement `Closeable` too?

Reply:
Nah because I proposed to remove init and close in these classes